pyblish_lite/app.py

```diff
@@ -52,9 +55,23 @@ def show(parent=None):
 
         ctrl = control.Controller()
 
-        win = window.Window(ctrl, parent)
-        win.resize(430, 600)
-        win.show()
+        win = None
+        try:
+            win = self._window
+            win.show()
+            win.activateWindow()
+            win.setWindowTitle(settings.WindowTitle)
+            print "Using existing window..."
+        except:
+            win = window.Window(ctrl, parent)
+            win.resize(430, 600)
+            win.setAttribute(QtCore.Qt.WA_DeleteOnClose)
```

Comment:
This isn't necessary, if you're doing it in the `__init__`, which I think you should.